bl/policy/policy_factory.go

```diff
@@ -37,13 +38,20 @@ func CreatePolicy(policies *cliClient.EvaluationPrerunPolicies, policyName strin
 
 	if policies != nil {
 		var chosenPolicy *cliClient.Policy
+		sawDefault := false
 
 		for _, policy := range policies.Policies {
-			if policyName == "" && policy.IsDefault {
+			if sawDefault && policy.IsDefault {
+				// If we are here, that means there are multiple policies
+				// set to default. So, throw an error
+
+				err = fmt.Errorf("invalid policy file: multiple policies are set to default")
+				return Policy{}, err
+			} else if policyName == "" && policy.IsDefault {
```

Comment:
if the user provides a policyName (`--policy` flag), this check will never detect the duplication because `policyName != ""`